### PR TITLE
Modern Spellbook fixes

### PR DESCRIPTION
**Pull request by DrHenny**
- Fixed issues with crossbow bolt enchanting not showing the correct sprite
- Removed all the old code overrides for the modern spellbook
- Fixed all issues with naming, levels, highlighting for teleport spells (mainly cache)
Cache Link: (MUST use this cache)
http://uppit.com/v1srlv0b4k8z

### CODE REVIEW

File: ElvargServer/src/main/java/com/elvarg/net/packet/impl/ButtonClickPacketListener.java [hunk 1]
```diff
@@ -143,8 +143,8 @@ public void execute(Player player, Packet packet) {
 			return;
 		}
 
-		if (player.getRights() == PlayerRights.DEVELOPER) {
-			player.getPacketSender().sendMessage("Button clicked: " + Integer.toString(button) + ".");
+		if (PlayerRights.isAdmin(player)) {
```
Comment (tobywisener) [inline ElvargServer/src/main/java/com/elvarg/net/packet/impl/ButtonClickPacketListener.java:146]: Logging is definitely a developer thing, not sure why you changed this. Its not an admin thing but if you really want it to be just do ```player.getRights() >= PlayerRights.ADMINISTRATOR```
(as owner and dev are above it in the enum)

File: ElvargServer/src/main/java/com/elvarg/game/model/Location.java [hunk 2]
```diff
@@ -396,4 +397,27 @@ public Location rotate(double degrees) {
...
+     * @param location
+     * @param radius
+     * @return
+     */
+    public static Location randomTile(Location location, int radius) {
+        Location chosen = null;
+        int requestedX = location.getX();
+        int requestedY = location.getY();
+        int height = location.getZ();
+        while(true) {
```
[the remark this answers is not in the document]
Comment (DrHenny) [reply ElvargServer/src/main/java/com/elvarg/game/model/Location.java]: k

**Comment by Mark7625** [comment]
You could remove widget debugging and add widget inspector from rl, also why did you remove the chat box teleports they are rlly nice tbh

**Comment by tobywisener** [comment]
> You could remove widget debugging and add widget inspector from rl, also why did you remove the chat box teleports they are rlly nice tbh

Keeping this PR open as I think you're working on new chat box teleports

**Comment by DrHenny** [comment]
cache no longer available from the link, will have to redo..

**Pull request closed without merging**